tests/tests.cpp

```diff
@@ -4,6 +4,33 @@
 
 #define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
 #include "catch.hpp"
+#include "image_buffer.h"
+#include "test_entry.h"
 
 // See https://github.com/catchorg/Catch2/blob/master/docs/tutorial.md for tutorial
 
+TEST_CASE("Image Buffer Behavior", "[image_buffer]") {
```

Comment:
these all pass. I need to setup CI on github so you can see that, but for now just trust me